Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -386,13 +453,26 @@ namespace AzToolsFramework
 
                     m_columnLayout->insertWidget(priorColumnIndex + 1, addedWidget, 0, widgetAlignment);
                 }
-
-                // insert after the found index; even if nothing were found and priorIndex is still -1,
-                // still insert one after it, at position 0
                 else
                 {
                     m_columnLayout->insertWidget(priorColumnIndex + 1, addedWidget);
                 }
+
+                //! If the sharePrior attribute is present and this propertyEditor is not next to a label,
+                //! add the previous widget to the column layout.
+                //! Set the SharePrior boolean so we know to create a new shared column layout, or add to an existing one
+                auto sharePrior = AZ::Dpe::Nodes::PropertyEditor::SharePriorColumn.ExtractFromDomNode(childValue);
+                if (sharePrior.has_value())
```

Comment:
I know it'd be pretty weird to put SharePriorColumn=false in a DOM, but it should technically work if it there. Could you change this to checking the boolean value instead of just checking for its presence?